doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -405,6 +410,11 @@ Update the Grub file
 
       $ sudo update-grub
```

Comment:
@fuzhongl , I don't think this is doing what we expect. Isn't that updating the Grub on the NVMe disk here while what we want is to upgrade Grub that's on the SATA disk (RTVM)?

Reply:
@deb-intel We'll wait until @gvancuts question to @fuzhongl is answered to merge and publish.

Reply:
@gvancuts I have a try today.You are right.
Please help to update with reboot from sda first; then everything is correct.
Thanks!